packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -203,10 +206,16 @@ class ForcePressGestureRecognizer extends OneSequenceGestureRecognizer {
 
   @override
   void addPointer(PointerEvent event) {
-    startTrackingPointer(event.pointer);
-    if (_state == _ForceState.ready) {
-      _state = _ForceState.possible;
-      _lastPosition = event.position;
+    // If the device has a maximum pressure of less than 1, (ie. on a non iOS
+    // device) we want do not want any callbacks to be called.
+    if (!(event is PointerUpEvent) && event.pressureMax <= 1.0) {
```

Comment:
pressureMax can't be < 1, so this can just be == 1 with an assert >= 1 above.
the comment doesn't match the code

Reply:
Couldn't it be less than one if that's what the manufacturers set for the max pressure?

Reply:
we define the minimum maximum pressure as 1.0 and the maximum minimum pressure as 1.0.

Reply:
Oh, ok!